### DISCUSSION TITLE
How to set default value based on selection

### DISCUSSION THREAD

**Discussion by apoorvmote** [react-hook-form/react-hook-form#4760] [Q&A] [answered]
https://codesandbox.io/s/react-hook-form-get-started-ts-forked-ed9k1?file=/src/index.tsx

In above code sandbox. I have names like the following
```ts
const persons = [
    {
      name: "jon",
      cat: "cat 1"
    },
    {
      name: "jane",
      cat: "cat 2"
    },
    {
      name: "doe",
      cat: "cat 3"
    }
  ];
```

Each person has default category and I want to automatically select category based on person selection.

And at the same time I want to give user option to override and choose different category than default category.

**Comment by bluebill1049** [comment, marked as answer]
use `setValue` api to update dependent input?

**Reply by apoorvmote** [reply]
Yes I just tried that and it worked. Thanks
